packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -80,6 +81,7 @@ import { setContext } from '../../shared/vscode/setContext'
 import { makeTemporaryToolkitFolder } from '../../shared'
 import globals from '../../shared/extensionGlobals'
 import { convertDateToTimestamp } from '../../shared/datetime'
+import { spawnSync } from 'child_process'
```

Comment:
use our ChildProcess wrapper unless there is a technical reason why it can't be used. https://github.com/aws/aws-toolkit-vscode/blob/8966edc82a33c5b7f00d45477941bcc333a10318/packages/core/src/shared/utilities/processUtils.ts#L70

It has logging, and will eventually gain some performance-tracking and telemetry features.